application/src/main/java/core/application/movies/repositories/CachedMovieRepositoryImpl.java

```diff
@@ -0,0 +1,57 @@
+package core.application.movies.repositories;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.stereotype.Repository;
+
+import core.application.movies.models.entities.CachedMovieEntity;
+import core.application.movies.repositories.mapper.CachedMovieMapper;
+import lombok.RequiredArgsConstructor;
+
+@Repository
+@RequiredArgsConstructor
+public class CachedMovieRepositoryImpl implements CachedMovieRepository {
+
+	private final CachedMovieMapper mapper;
+
+	@Override
+	public CachedMovieEntity saveNewMovie(CachedMovieEntity movie) {
+		return mapper.save(movie);
+	}
```

Comment:
`MyBatis` 의 insert 의 return 은 `int` 인 것으로 알고 있습니다.
그래서 `saveNewMovie` 를 살짝 수정해야 될지도 모르겠네요.

이런 식으로?

```suggestion
	@Override
	public CachedMovieEntity saveNewMovie(CachedMovieEntity movie) {
	        int insertionResult = mapper.save(movie);
		return movie;
	}
```